README.md

```diff
@@ -97,7 +97,7 @@ export const runtime = "nodejs"; // default
 /** @type {import('next').NextConfig} */
 const nextConfig = {
   experimental: {
-    serverComponentsExternalPackages: ["pdf-parse"], // Puts pdf-parse in actual NodeJS mode with NextJS App Router
+    serverComponentsExternalPackages: ["pdf-parse", "@gxl/epub-parser"], // Puts pdf-parse and @gxl/epub-parser in actual NodeJS mode with NextJS App Router
```

Comment:
since I cannot test nextjs app, so I just copy like pdf-parse config here.
Advice needed. thank you

Reply:
I don't think you necessarily need to put this here, but I'll give it a try with the NextJS configuration and see what happens.

`create-llama` is probably your best bet to test this. @marcusschiesser thoughts?

Reply:
@m1911star and @yisding 
yes, in `create-llama` we're just using:
```
serverComponentsExternalPackages: ["llamaindex"]
```
This should work here, too, but explicitly enumerating the problematic dependencies (before just "pdf-parse" and now "@gxl/epub-parser") has the advantage that we know who is the culprit and what we have to do.
This brings us to the question: @m1911star, is there another epub parser package that works with nextjs?
There seem to be alternatives: https://www.npmjs.com/search?q=epub-parser

Reply:
I have dived into alternatives of @gxl/epub-parser and did not find the right choice suitable for next app.

Also `@gxl/epub-parser` can not work in next app.

Im working to find another solution. Thanks.

Reply:
Thanks @m1911star! Appreciate the contribution!